docs/docker.md

```diff
@@ -15,24 +16,74 @@ You want a dockerfile that is structured something similar to this
 ```Dockerfile
 # Dockerfile
 
-# Build container
-FROM continuumio/miniconda:latest as conda
+# -----------------
+# Builder container
+# -----------------
+FROM continuumio/miniconda:latest as builder
 
 ADD conda-linux-64.lock /locks/conda-linux-64.lock
```

Comment:
I believe best practice is to prefer `COPY` over `ADD` when not using `ADD` features.

```suggestion
COPY conda-linux-64.lock /locks/conda-linux-64.lock
```